common/python/generate_app.py

```diff
@@ -158,6 +165,7 @@ def implement_blocks(self, ini, path, type):
                 # for carrier block
                 block = BlockConfig(section, type, number, ini_path, siteNumber)
                 block.register_addresses(self.counters)
+                block.generateCalcExtensions()
```

Comment:
Could we use a consistent naming convention for functions? I think `generate_calc_extensions()` would be more pythonic